site/content/blog/2023-09-25-postman-add-on/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+images:
+- /blog/2023-09-25-postman-add-on/images/zapbot-postman.png
```

Comment:
This needs to have the full URL or it wont work as a social preview image

Reply:
Done